libarchive/archive_write_set_format_pax.c

```diff
@@ -401,6 +405,53 @@ archive_write_pax_header_xattrs(struct archive_write *a,
 	return (ARCHIVE_OK);
 }
 
+static void
+archive_write_pax_header_kw(struct pax *pax, const char *encoded_name,
+    const void *value, size_t value_len)
+{
+	if (pax->flags & WRITE_GENERIC_KW) {
+		/* only supporting string values, that is what the generic standard allows */
```

Comment:
this comment LIES. we support binary, as that allows the other binary value attributes as well.